src/core/src/bootstrap/Constants.py

```diff
@@ -160,6 +160,7 @@ class AutoAssessmentStates(EnumBackport):
     MAX_ASSESSMENT_RETRY_COUNT = 5
     MAX_INSTALLATION_RETRY_COUNT = 3
     MAX_IMDS_CONNECTION_RETRY_COUNT = 5
+    MAX_REPO_REFRESH_RETRY_COUNT = 3
```

Comment:
MAX_ZYPPER_REPO_REFRESH_RETRY_COUNT  since we are only doing it for one.

Reply:
Done

Reply:
Set this to 5 too

Reply:
Done